### DISCUSSION TITLE
is there any way to add emails to skiovox

### DISCUSSION THREAD

**Discussion by DWPBEG** [3kh0/ext-remover#841] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
The title says it all

**Comment by 3kh0** [comment]
Install the extension

**Reply by Aka-but-nice** [reply, marked as answer]
or simply visit https://tinyurl.com/addsession :moyai:

**Reply by Epicminer256** [reply]
With skiovox-helper, the top right of the newtab page has a add account button.

**Reply by akabutnicer** [reply]
or simply visit https://tinyurl.com/addsession 🗿
